util/config_manager.cc

```diff
@@ -202,8 +207,10 @@ std::unique_ptr<Config> DefaultConfig(bool notify) noexcept {
       // enable main but not subscriptions yet (need clusters in main account)
       true, false,
       // do not dump main or subs
-      false, false, kDefaultVerbosity, log_num_size.max_size,
-      log_num_size.max_files, get_default_common_tags());
+      false, false, kDefaultVerbosity,
+      // do not send metrics in parallel to reduce memory footprint used
```

Comment:
Ballpark what is the memory overhead? How much should we be concerned about it?

Reply:
I was thinking mostly about sending 1M metrics or more. Right now we'd only consume memory for converting one batch (10k measurements) to JSON, and compressing that. With the parallel approach we'd use 100x more memory. It might not be an issue since usually if you're sending so many measurements per-minute that is the actual job of the application. The normal use case wouldn't be affected.